### ISSUE TITLE
Caps Lock prevents Windows Key from working

### ISSUE THREAD

**Issue by augster** [symless/synergy#5951]
### Operating Systems ###

Server: Windows 10
Client: Windows 10

### Synergy Version ###

1.8.8

### Steps to reproduce bug ###

- I have a mouse and keyboard connected to my Server that is shared with my surface pro 3. When I have Caps Lock enabled on my client, the Windows Key stops working until Caps Lock is turned off. 
- Other keys are also broken in other programs. I.e., When Caps Lock is On, the Delete key doesn't work in Autocad (2013 version). Works fine when caps lock is off though. I've tried changing "halfDuplexCapsLock = true" on both client and server but that doesn't fix the problem.

**Comment by augster** [comment]
So I've discovered more about why this is happening. When I press Caps Lock on my server keyboard, the Caps Lock on my client is not toggled. Though, when I type letters, each key press briefly presses my Caps Lock and corresponding key(to simulate caps lock). So capital letters get typed but Win-Key does not respond since it essential becomes Caps Key + Win-Key. I tried mapping Hotkeys to my Caps Lock but that doesn't help either. Pretty annoying.
